core/templates/components/common-layout-directives/common-elements/background-banner.component.html

```diff
@@ -0,0 +1,3 @@
+<div style="position: absolute; width: 100%; background-color: #aed2e9; text-align: center; overflow: hidden;">
+  <img [src]="bannerImageFileUrl" align="center" style="width: 3000px; top: 65px; max-width: 3000px;" alt="">
```

Comment:
@bansalnitish Instead of deleting then a new-file, can we do rename and make changes in the same file (not sure whether it's possible)? [This is just to save the git history of the file.]

You can use `git mv`.

Reply:
Hm, I think you can use the `git log --follow /path/to/file` for that (see [doc](https://git-scm.com/docs/git-log), SO [comment](https://stackoverflow.com/a/2314757)).

Also, from Git [FAQ](https://git.wiki.kernel.org/index.php/Git_FAQ#Why_does_Git_not_.22track.22_renames.3F):

> Git has a rename command git mv, but that is just for convenience. The effect is indistinguishable from removing the file and adding another with different name and the same content.

Reply:
Let's do this for future PRs. I think doing it for this one retroactively might be tricky, and we need to get it in to provide an example for Angular migration of directives.

Reply:
Oh wait, scratch the previous comment, I misunderstood. I think the "git log" is more for the person who is reading, rather than the person who is committing, so it looks like there's no action for @bansalnitish here.